kyuubi-main/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -159,4 +159,11 @@ object SparkProcessBuilder {
   private final val PROXY_USER = "--proxy-user"
   private final val PRINCIPAL = "spark.kerberos.principal"
   private final val KEYTAB = "spark.kerberos.keytab"
+  private final val SPARK_SUBMIT_FILE = {
+    if (String.valueOf(System.getProperty("os.name")).toLowerCase().contains("win")) {
```

Comment:
We can add some thing like below in `kyuubi-common` `Utils` object
```scala
import org.apache.commons.lang3.SystemUtils.IS_OS_WINDOWS

...

/**
 * Whether the underlying operating system is Windows.
 */
val isWindows = SystemUtils.IS_OS_WINDOWS

```

If we want to support win, we can make such a step for further update. WDYT

Reply:
I agree. It's better.

Reply:
can you make a change in this way?

Reply:
OK